ci-operator/config/ComplianceAsCode/compliance-operator/ComplianceAsCode-compliance-operator-master.yaml

```diff
@@ -128,6 +128,13 @@ tests:
     - as: test-ocp4-pci-dss-node
       cli: latest
       commands: make e2e-rosa
+      dependencies:
+      - env: IMAGE_FROM_CI
+        name: compliance-operator
+      - env: CONTENT_IMAGE_FROM_CI
+        name: testcontent
+      - env: OPENSCAP_IMAGE_FROM_CI
+        name: testopenscap
```

Comment:
The `CONTENT_IMAGE_FROM_CI` and `OPENSCAP_IMAGE_FROM_CI` are not used in any of the rosa tests yet, are they?

Reply:
no, I don't think so